ci/full-dependencies/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# PANOPTES development container
+# Build with:
+#
+#         docker build --tag panoptes/build --file $POCS/ci/Dockerfile-3 -- $POCS
+#
+# This makes $POCS the "context" for the build, i.e. the root folder
+# from where files such as requirements.txt may be copied.
+
+FROM ubuntu:bionic as build-env
+
+LABEL description="PANOPTES Development Container"
+LABEL author="Developers for PANOPTES project"
+LABEL url="https://github.com/panoptes/POCS"
+
+ARG pan_dir=/var/panoptes
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV ENV /root/.bashrc
+ENV SHELL /bin/bash
+ENV PANDIR $pan_dir
+ENV PANLOG $PANDIR/logs 
+ENV POCS $PANDIR/POCS  
+ENV PAWS $PANDIR/PAWS  
+ENV PANUSER root
+ENV SOLVE_FIELD=/usr/bin/solve-field
+
+################################################################################
+# Several update steps before our custom commands. By rarely changing these
+# commands, we allow the caching that 'docker build' performs to save us some
+# time when iterating on the Panoptes specific steps (i.e. instead of executing
+# these, the cached image layer is used).
+
+# Update the information we know about package versions.
+RUN apt-get update --fix-missing
```

Comment:
My understanding is that in general you want to minimize the number of `RUN`, `COPY`, and `ADD` steps as they each create new layers and thus inflate the size. I realize we might be caching them but it still causes the overall container size to be larger.

Unless this is being used as part of a multi-stage build? But if that is the case it seems like we should stage it up even more.